models/notification.go

```diff
@@ -124,6 +126,18 @@ func (message *Notification) checkConfig() {
 	message.Recipients = newRecipient
 }
 
+func cleanNotificationDescription(content string) string {
+	newContent = strings.ReplaceAll(content, "\n", "")
```

Comment:
`newContent` is not declared before it’s used. You should declare the `newContent` variable first, like this:

```go
newContent := strings.ReplaceAll(content, "\n", "")
```

Reply:
This is fixed in the second commit.